rootfs/etc/deis-dockerbuilder.yaml

```diff
@@ -11,14 +11,18 @@ spec:
   containers:
     - name: repo_name
       imagePullPolicy: Always
-      image: smothiki/dockerbuilder:5.5
+      image: arschles/dockerbuilder:5.5
       env:
         - name: DEBUG
           value: "1"
         - name: TAR_URL
           value: tar-url
-        - name: IMAGE
+        - name: IMG_NAME
           value: imagename
+        - name: ACCESS_KEY_FILE
+          value: /var/run/secrets/object/store/access_key
+        - name: ACCESS_SECRET_FILE
+          value: /var/run/secrets/object/store/access_secret
```

Comment:
No need to specify access key file and access secret 
They are the volumes that needs to be mounted. and use k8s secrets to define them